umbraco-cloud/getting-started/the-umbraco-cloud-portal/organizations.md

```diff
@@ -1,15 +1,18 @@
 # Organizations
 
-On Umbraco Cloud, you can get an Umbraco Cloud Organization. With an Organization, you get an overview of all the projects that have been created by you and the members of your Organization.
+On Umbraco Cloud it is possible to get an Organization. An organization is handy if you are a partner or are managing a lot of projects for different customers.
 
-If you are interested in getting an Organization, please reach out to our Support Team through the Chat box in your project overview.
-
-## Managing your Organization
+With an organization you get an overview of all projects and members part of the organization and you manage project payments and other options.
 
 In the following sections, we will go through the different options that are available in an Organization:
 
 ![Organization Overview](images/Cloud-org-overview.png)
 
+{% hint style="info" %}
+If you are interested in getting an Organization, please reach out to our Support Team through the Chat box in your project overview.
+{% endhint %}
```

Comment:
I found out that if you get the warning that you need to use less than 25 words in an hint note, but you actually do use less than 25 words, then if you add a space before and after the hint the warning will be gone 😁 
```suggestion
{% hint style="info" %}

If you are interested in getting an Organization, please reach out to the Support Team through the Chat box in your [project overview](https://www.s1.umbraco.io/projects).

{% endhint %}
```